src/wallet3/db_schema.cpp

```diff
@@ -47,11 +49,15 @@ namespace wallet
             last_scan_height INTEGER NOT NULL DEFAULT -1,
             scan_target_hash TEXT NOT NULL,
             scan_target_height INTEGER NOT NULL DEFAULT 0,
-            output_count INTEGER NOT NULL DEFAULT 0
+            output_count INTEGER NOT NULL DEFAULT 0,
+            spend_priv BLOB,
+            spend_pub BLOB,
+            view_priv BLOB,
+            view_pub BLOB
           );
```

Comment:
This table structure is more than a little unwieldy, and is going to get worse as we add more things.  I think we need to redesign it to be Nx1 instead of 1xN, with something like:

```SQL
CREATE TABLE state (
    id TEXT PRIMARY KEY NOT NULL,
    val_numeric INT,
    val_binary BLOB,
    val_text TEXT,
    -- Exactly one val_* must be set:
    CHECK((val_numeric IS NOT NULL) + (val_binary IS NOT NULL) + (val_text IS NOT NULL) == 1)
) STRICT;
```

where each field stored becomes a `(id, val_numeric)`, `(id, val_binary)`, or `(id, val_text)` depending on what is being stored.